### ISSUE TITLE
About experiment results questions

### ISSUE THREAD

**Issue by DuTim** [Sahandfer/CEM#3]
Hi , Sahandfer!
    I read your paper carefully, but I have some questions about the reproduction of your experimental results. In the automatic evaluation experiment, I completely configured according to your experimental environment, but there is some gap between my running results (ACC, PPL) and those provided by you. 

What adjustments do I need to make to the Settings to achieve the effect in your paper?  Hyperparameter, environment configuration? 

Thank you very much 

environment setting: 
![image](https://user-images.githubusercontent.com/50939742/158541648-469360fb-e926-4eb5-82de-8bdec9778771.png)
myself results:
![image](https://user-images.githubusercontent.com/50939742/158541766-a217b02e-7869-4be7-a3e9-d0dd0be2c246.png)

**Comment by Sahandfer** [comment]
Dear @DuTim, 
I had made a mistake in making the requirements.txt file. The environment for my run was running python 3.8.5 with CUDA 10.2 and PyTorch 1.8. I have corrected the requirements in the latest commit. 
In addition, I was also made aware that training on different systems has produced different results, which might suggest that I also made a mistake when setting the seed. However, I have yet not figured out how to fix this issue. 
To address the problem of reproducibility, I have uploaded the used checkpoint for the results in the paper. You can run the following command to generate the responses using the provided checkpoint: python main.py --model cem --test --model_path save/CEM_19999_41.8034. You can also refer to [results.txt](https://github.com/Sahandfer/CEM/files/8308531/results.txt). 
Thank you for your interest in our work and I apologize for the inconvenience caused.
